apps/smart-contracts/core/test/Collateral.test.ts

```diff
@@ -198,10 +188,10 @@ describe('=> Collateral', () => {
   })
 
   describe('# setManager ', () => {
+    snapshotter.usesCustomSnapshot()
```

Comment:
```suggestion
    snapshotter.usesCustomSnapshot()
```
Are we not naming these anymore? i.e. `Collateral-setManager`, this is being done in 674 no?